acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -38,19 +38,30 @@ const (
 func TestAPIGateway_Basic(t *testing.T) {
 	cases := []struct {
 		secure bool
+		psa    bool
```

Comment:
Curious why the variable name changed here compared to the original PR into `main`